benchmarks/media-streaming/dataset/Dockerfile

```diff
@@ -1,18 +1,24 @@
-FROM ubuntu:14.04
+FROM cloudsuite/debian:base-os
 MAINTAINER Alexandros Daglis <alexandros.daglis@epfl.ch>
 
-RUN apt-get update -y && \
-	apt-get install -y --force-yes software-properties-common && \
-	apt-add-repository "ppa:mc3man/trusty-media" && \
-	add-apt-repository ppa:deadsnakes/ppa && \
-	apt-get update -y && \
-	apt-get install -y --no-install-recommends \
+RUN apt-get update -y \
+	&& apt-get install -y --no-install-recommends \
+	software-properties-common \
 	build-essential \
-	python3.5 \
-	ffmpeg \
-  curl \
+	curl \
+	wget \
+	yasm \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN wget --progress=bar:force -O /tmp/ffmpeg-4.1.5.tar.bz2 http://ffmpeg.org/releases/ffmpeg-4.1.5.tar.bz2 \
```

Comment:
Why not get `ffmpeg` using `apt-get`?
Compiling it takes a long time, particularly on arm and risc-v as they are being emulated when building.

Reply:
There is no ffmpeg for riscv, so we have installed it from source instead
https://packages.debian.org/buster/ffmpeg

Reply:
This makes it impossible to build on travis.

Reply:
OK so there are two solutions in my opinion, either we drop the risc-v support for this workload for now, or we build using the source only for risc-v. I think you can handle this by using the parameter `$TARGETPLATFORM` inside the dockerfile and decided what to do based on the platform. Can you try this?

Reply:
(We are going to hit this risc-v wall several other times probably for other workloads, so we need to decide what we can we about it.)

Reply:
I ran the buildx command for dataset on baremetal instance (nproc --all = 48), it takes ~28mins to run for all 3 architectures combined. (Travis has only 2 cores)
Option1: Let the Travis build only for amd64 and we can manually push the image.
Option2: Have conditional install based on the platform
```
RUN set -ex \
	&& case "$TARGETPLATFORM" in \
	# riscv64 debian sid
	linux/riscv64) \
            # install from source
        *) \
            # install using apt-get
```
I guess in both cases we cannot build for risc-v on travis, as it takes more than the travis time limit.

Reply:
Yeah, I also tried it locally and it took around half an hour to complete, but things were running in parallel. On Travis, the build are done sequentially as there are not enough cores. My hope is that if we only build from source for risc-v, the other two will finish quickly and there is more time for the risc-v build to complete, don't know how long it's gonna take on Travis though.

Here is what I suggest:
* remove risc-v for this benchmark for now so we can merge this and move on
* create another PR that tries to add risc-v based on what we discussed above (perhaps we need to change the time limit to 50, or more if possible)

Reply:
You forgot to add ARM.

Reply:
Updated